src/Featurizers/StandardScaleWrapperFeaturizer.h

```diff
@@ -0,0 +1,331 @@
+///                 and the `StandardDeviationEstimator` to create a `StandardScalerTransformer` object.
+///
+template <
+    typename InputT,
+    typename TransformedT,
+    size_t MaxNumTrainingItemsV=std::numeric_limits<size_t>::max()
+>
+class StandardScaleWrapperEstimatorImpl : public TransformerEstimator<InputT, TransformedT> {
+public:
+    // ----------------------------------------------------------------------
+    // |
+    // |  Public Types
+    // |
+    // ----------------------------------------------------------------------
+    using BaseType                          = TransformerEstimator<InputT, TransformedT>;
+    using TransformerType                   = StandardScalerTransformer<InputT, TransformedT>;
+
+    // ----------------------------------------------------------------------
+    // |
+    // |  Public Methods
+    // |
+    // ----------------------------------------------------------------------
+    StandardScaleWrapperEstimatorImpl(AnnotationMapsPtr pAllColumnAnnotations, size_t colIndex, bool withMean, bool withStd);
+    ~StandardScaleWrapperEstimatorImpl(void) override = default;
+
+    FEATURIZER_MOVE_CONSTRUCTOR_ONLY(StandardScaleWrapperEstimatorImpl);
+
+private:
+    // ----------------------------------------------------------------------
+    // |
+    // |  Private Data
+    // |
+    // ----------------------------------------------------------------------
+    size_t const                            _colIndex;
+    // TODO: change bools to bit flags
+    bool   const                            _withMean;
+    bool   const                            _withStd;
+
+    // ----------------------------------------------------------------------
+    // |
+    // |  Private Methods
+    // |
+    // ----------------------------------------------------------------------
+    bool begin_training_impl(void) override;
+    FitResult fit_impl(typename BaseType::InputType const *, size_t) override;
+    void complete_training_impl(void) override;
+
+    // MSVC has problems when the definition is separate from the declaration
+    typename BaseType::TransformerUniquePtr create_transformer_impl(void) override {
+        // ----------------------------------------------------------------------
+        using StandardStatisticalAnnotationData = Components::StandardStatisticalAnnotationData<InputT>;
+        using StatisticalMetricsEstimator       = Components::StatisticalMetricsEstimator<InputT, MaxNumTrainingItemsV>;
+
+        using StandardDeviationAnnotationData   = Components::StandardDeviationAnnotationData;
+        using StandardDeviationEstimator        = Components::StandardDeviationEstimator<InputT, MaxNumTrainingItemsV>;
+        // ----------------------------------------------------------------------
+        // if _withMean is false, average is set to 0
+        std::double_t average = 0;
+        // if _withStd is false, deviation is set to 1
```

Comment:
Why do you set this to 1 if false, but set the average to 0 if its false?

Reply:
Because if withStd and withMean are set to false, default operation is to scale data centered at 0 with unit standard deviation. So average is set to 0 and standard deviation is set to 1.